src/Fields.php

```diff
@@ -731,7 +731,7 @@ protected function wf_crm_get_fields($var = 'fields') {
       if (isset($sets['membership'])) {
         $fields['membership_membership_type_id'] = [
           'name' => t('Membership Type'),
-          'type' => 'civicrm_select',
+          'type' => 'select',
           'expose_list' => TRUE,
```

Comment:
Ah 🎉... ok this totally fixes things 

Quick search in `Fields.php` -> yields no other accidental `civicrm_select` usage.